.dependabot/config.yml

```diff
@@ -0,0 +1,12 @@
+version: 1
```

Comment:
A `version: 2` of the config has been released by the `dependabot` team since preview is over. See [this](https://help.github.com/en/github/administering-a-repository/enabling-and-disabling-version-updates) for updated config syntax. 

I personally use the following: 

```
version: 2
updates:
  # Enable version updates for npm
  - package-ecosystem: 'npm'
    # Look for `package.json` and `lock` files in all directories
    directory: '/'
    # Check the npm registry for updates every day (weekdays)
    schedule:
      interval: 'daily'

  - package-ecosystem: 'github-actions'
    directory: '/'
    # Check for updates once a week
    schedule:
      interval: 'weekly'
```

Reply:
Yes, we should go with version 2. Also I'm supporting updates to be checks weekly because the new version does not support auto merging anymore.